### PR TITLE
Add PayPal logos currently in storefront :/ :/ :(

### PR DESCRIPTION
**Pull request by edwardsfriedman**

### CODE REVIEW

File: lib/storefront/paypal-button-logo.svg [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="93.8px"
+	 height="25px" viewBox="0 0 93.8 25" style="enable-background:new 0 0 93.8 25;" xml:space="preserve">
+<title>Paypal</title>
+<desc>Paypal Logo</desc>
+<g id="Layer_1">
+	<g>
+		<g>
+			<g>
+				<g>
+					<path style="fill:#009BDE;" d="M63.3,0H56c-0.5,0-0.9,0.4-1,0.9l-3,18.7c-0.1,0.4,0.2,0.7,0.6,0.7h3.7c0.3,0,0.6-0.3,0.7-0.6
```
Comment (edwardsfriedman) [inline lib/storefront/paypal-button-logo.svg:12]: fills are necessary on this one, it's multi-color

File: lib/storefront/paypal.svg [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="286px"
+	 height="76.2px" viewBox="0 0 286 76.2" style="enable-background:new 0 0 286 76.2;" xml:space="preserve">
+<title>Paypal</title>
+<desc>Paypal Logo</desc>
```
Comment (ckeisato) [inline lib/storefront/paypal.svg:6]: can you give this one a different title/desc from the one above?

Comment (edwardsfriedman) [reply lib/storefront/paypal.svg:6]: Lol no they're the same except one has color! Which is not helpful for someone who can't see it anyway.

Comment (ckeisato) [reply lib/storefront/paypal.svg:6]: can we add it to the desc?

**Pull request merged**